tests/Unit/Framework/SetupLocalPythonEnvironment.cpp

```diff
@@ -26,8 +26,10 @@ void prepend_python_path(const std::string& new_path) {
     ERROR_NO_TRACE("Trying to add path '"
                    << new_path
                    << "' to the python environment during setup "
-                      "but this directory does not exist. Maybe "
-                      "you have a typo in your path?");
+                      "but this directory does not exist. Either "
+                      "you have a typo in your path, or you need "
+                      "to create the appropriate site-packages "
+                      "directory after upgrading python.");
```

Comment:
Maybe just delete this error message altogether? Should be ok to add nonexistent directories to the path.

Reply:
I assume someone put in the current message because they had a typo in their path and it led to confusion.  So I'd prefer to keep it